docs/installation.md

```diff
@@ -0,0 +1,118 @@
+# Installation
+
+## Table Of Contents
+
+- [Requirements](#requirements)
+- [Hyperconverged Cluster Operator (HCO) Installation](#hyperconverged-cluster-operator-hco-installation)
+- [Manually Installation](#manually-installation)
+- [Building](#building)
+
+## Requirements
+
+SSP operator requires an OpenShift cluster in order to deliver complete functionality.
```

Comment:
Running it on Kubernetes with a reduced set of features is still possible though.

Reply:
Yes, but it doesn't deliver complete functionality.